frontend/src/citizen-frontend/decisions/decisions-page/Decisions.tsx

```diff
@@ -69,7 +69,7 @@ export default React.memo(function Decisions() {
                 (decision) =>
                   'decisionMade' in decision
                     ? decision.decisionMade?.formatIso()
-                    : decision.sentDate.formatIso()
+                    : [decision.sentDate.formatIso(), decision.type]
```

Comment:
What happens if there are mixed strings and arrays here? Should the `decisionMade` timestamp also be wrapped in an array for the sorting to work?

Reply:
Never mind, seems to work correctly 👍